src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,11 @@
 package mate.academy.lib;
 
-public @interface Inject {
```

Comment:
Remove the empty line between the package declaration and the import statements to keep the code clean and consistent.